src/metrics.rs

```diff
@@ -398,6 +398,10 @@ pub struct GooseRequestMetricAggregate {
     pub load_test_hash: u64,
     // Counts requests per second. Each element of the vector represents one second.
     pub requests_per_second: Vec<u32>,
+    // Counts errors per second. Each element of the vector represents one second.
+    pub errors_per_second: Vec<u32>,
+    // Maintains average response time per second. Each element of the vector represents one second.
+    pub average_response_time_per_second: Vec<(u32, f32)>,
```

Comment:
`response_time` is u64 and we want a floating point number here since we are calculating an average. We could use f64, but I don't think that we need as much precision and we would still need to cast from u64 to f64?